components/atoms/Button/index.jsx

```diff
@@ -1,10 +1,16 @@
-import './styles.module.scss';
+import React, {useState} from 'react';
+import styles from './styles.module.scss'; 
+
+const Button = ({size, disabled, children}) => {  
+
+  const btnSize = size === "lg" ? styles.lg : size === "md" ? styles.md : size === "sm" ? styles.sm : ""
 
-const Button = () => {
   return (
-    <>
-      <p>Button</p>
-    </>
+    <button 
+      className={`${bttnSize} ${styles.bttn}`} 
```

Comment:
No creo que ande solo pasandole bttnSize, lo que hay que pasarle es la propiedad del modulo scss que corresponde, por lo tanto se puede usar styles[bttnSize], para que sea dinamico
```suggestion
      className={`${styles[bttnSize]} ${styles.bttn}`} 
```

Reply:
el btnSize es el nombre de la variable que recibe el size desde el padre

Reply:
ahhh ya entendi, esta bueno como lo implementaste!